### DISCUSSION TITLE
Direct fan control

### DISCUSSION THREAD

**Discussion by Arvilsy** [seerge/g-helper#2852] [Q&A] [answered]
Hello, so ive been using ghelper 0.176 with the experimental fan control and it works great. i was wondering how to get this feature on higher versions like 0.179. is there a specific code or build or something to make that option appear ?

**Comment by seerge** [comment, marked as answer]
@Arvilsy hello. Build in that thread is also updated https://github.com/seerge/g-helper/discussions/2272 :) Just download latest version there
